src/LibraryManager.Build/Microsoft.Web.LibraryManager.Build.csproj

```diff
@@ -16,6 +16,8 @@
     <PackageReference Include="Microsoft.Build.Framework" PrivateAssets="all" />
     <PackageReference Include="Microsoft.Build.Tasks.Core" PrivateAssets="all" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.IO.Redist" Condition="'$(TargetFramework)' == '$(NetFxTFM)'" />
```

Comment:
Are these being added as a means to pin the new versions?  Could we instead set CentralPackageTransitivePinningEnabled in Directory.Packages.props so that transitive references will respect the version specified there, but we're not adding new primary references?  (For example, if the dependency tree changes, adding this as a primary package means we'll always reference it even if it's no longer used.)

Reply:
(Trying it out)

Reply:
Double checked and all is well.